src/netbase.cpp

```diff
@@ -58,7 +59,17 @@ std::vector<CNetAddr> WrappedGetAddrInfo(const std::string& name, bool allow_loo
     addrinfo* ai_res{nullptr};
     const int n_err{getaddrinfo(name.c_str(), nullptr, &ai_hint, &ai_res)};
     if (n_err != 0) {
-        return {};
+        if ((ai_hint.ai_flags & AI_ADDRCONFIG) == AI_ADDRCONFIG) {
```

Comment:
looks like this can fix our IPv6 on CI: https://github.com/dashpay/dash/pull/6038

I will test it

Reply:
it doesn't: https://gitlab.com/dashpay/dash/-/jobs/8212452793